soh/src/code/code_80097A00.c

```diff
@@ -27,7 +27,7 @@ u16 gUpgradeCapacities[][4] = {
     { 0, 20, 30, 40 },     // Bomb Bags
     { 0, 0, 0, 0 },        // Unused (Scale)
     { 0, 0, 0, 0 },        // Unused (Strength)
-    { 99, 200, 500, 500 }, // Wallets
+    { 99, 200, 500, 999 }, // Wallets
```

Comment:
interesting that there were 2 500 rupee wallets in this before

Reply:
Yup, not sure if they originally planned a 999 wallet or if they just needed that 4th upgrade slot for the other capacity upgrades that are also handled by this code, but thankfully it makes those fairly easy to implement.